### PR TITLE
:sparkles: add team introduction link to README.md

### PR DESCRIPTION
**Pull request by hyxrxn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,9 @@
 - 현지에서 구하기 어려운 재료를 대체할 수 있는 대체 재료 제안
 - 조리 단계별 레시피를 영상과 간략한 글을 통해 제공
 - 조리 시간에 따른 레시피 제공 속도 조절
+
+### 팀원 소개
+- <a href="https://github.com/oshyun00/github-test/blob/dev/ella/ella.md">엘라</a>
+- <a href="https://github.com/oshyun00/github-test/blob/dev/crong/crong.md">악어</a>
+- <a href="https://github.com/oshyun00/github-test/blob/dev/pond/pond.md">폰드</a>
+- <a href="https://github.com/oshyun00/github-test/blob/dev/atto/atto.md">아토</a>
```
Comment (oshyun00) [inline README.md:14]: ?

**Review by oshyun00 — COMMENTED** [review]
?

File: README.md (hunk 1 above)
Comment (Hogu59) [reply README.md:14]: 그러게요ㅋㅋ

Comment (hyxrxn) [reply README.md:14]: 너무 잘했네요

Comment (Hogu59) [reply README.md:14]: 확실해?

**Pull request merged**